chipsec/utilcmd/uefi_cmd.py

```diff
@@ -178,17 +178,11 @@ def requires_driver(self):
 
         parser.parse_args(self.argv[2:], namespace=self)
 
-        # No driver required when printing the util documentation
-        if len(self.argv) < 3:
+        # No driver for decode functionality
+        if self.argv[2] in ('decode'):
```

Comment:
Did you want in ['decode']? Currently if self.argv[2] = "d" this will return True.